realm/realm-library/src/main/java/io/realm/StandardRealmSchema.java

```diff
@@ -110,7 +110,7 @@ public RealmObjectSchema create(String className) {
         if (realm.getSharedRealm().hasTable(internalTableName)) {
```

Comment:
And `LangBindHelper::add_table()` will probably throw and we will eventually see it here.